code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -99,9 +99,12 @@
 /obj/structure/constructshell/attackby(obj/item/O, mob/user, params)
 	if(istype(O, /obj/item/device/soulstone))
 		var/obj/item/device/soulstone/SS = O
+		if(!iscultist(user) && !iswizard(user))
```

Comment:
Core it doesn't even work like that anymore you scrub. 

If you make a construct as a non-cultist, its still gonna be slaved to nar-sie and murder crew. There's no reason to have the kind of interaction.